arguments.go

```diff
@@ -27,15 +27,31 @@ import (
 	"io/ioutil"
 )
 
+// ArgReader is the interface for the arg2 and arg3 streams on an
+// OutboundCallResponse and an InboundCall
+type ArgReader interface {
+	io.ReadCloser
+}
```

Comment:
Optionally terser as:

```
type ArgReader io.ReadCloser
```

If you're planning to add methods to this interface in a follow-up PR, though, definitely makes sense to keep it as-is.